### DISCUSSION TITLE
VLAN's Not Importing Into Correct VLAN Group

### DISCUSSION THREAD

**Discussion by KPTheProf** [bb-Ricardo/netbox-sync#372] [General]
Hi

I have a large number of VLAN's and multiple VLAN Groups configured in Netbox.
I have a VLAN Group for VMware VLAN's but when I import the data, it is imported into the same VLAN number but in a different group.
Is there a way to configure the settings.ini to specify which VLAN group I want the VLAN's added to or updated?

**Comment by bb-Ricardo** [comment]
Hi, this is currently not supported.

It would be great if you could open a feature request and describe how you would organise it in the config.

Thank you
